### PR TITLE
Add links for sponsors

### LINKED ISSUE #41: add links to sponsors
and making the sponsor logos clickable

### PR DESCRIPTION
**Pull request by srijonsaha**
closes #41

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -1,21 +1,21 @@
 /**
  * This file contains configuration for most aspects of the website that might need frequent changing.
- * 
+ *
```
Comment (bobheadxi) [inline src/config.ts:3]: nit: is there an eslint rule for this? i'm not seeing any warnings for trailing whitespaces

File: src/sections/Sponsors.vue [hunk 2]
```diff
@@ -7,13 +7,15 @@
...
         <div v-for="(col, i) in columns" :key="'column-'+i" class="tile is-vertical sponsor-column">
           <div v-for="(s, j) in col" :key="'row-'+i+'-'+j" class="tile sponsor-container">
-            <img ref="sponsor-logo"
-              :src="s.logoURL"
-              :alt="s.name"
-              :style="{
-                filter: s.logoFilter,
-                'object-fit': 'contain',
-              }" />
+            <a :href="s.website" target="_blank">
```
Comment (bobheadxi) [inline src/sections/Sponsors.vue]: unfortunately the additional nesting has changed the alignment of the logos ( ͡° ʖ̯ ͡°)

[before](https://ubclaunchpad.github.io/new#sponsors):

image

after:

image

the logo now floats to the top, when they should be vertically centered

File: src/config.ts (hunk 1 above)
Comment (bobheadxi) [reply src/config.ts:3]: if there is, let's just add it to `package.json`'s eslint rules

Comment (srijonsaha) [reply src/config.ts:3]: My VS Code settings auto trims whitespace. I'll add the eslint rule: https://eslint.org/docs/rules/no-trailing-spaces.

**Comment by bobheadxi** [comment]
uh a hit merge but it looks like the PR is still here, even though the commit is in??

https://github.com/ubclaunchpad/new/commit/d4b3b29fad6729357e08010258db361f20ba613a

**Comment by bobheadxi** [comment]
This PR was merged in d4b3b29 🤷‍♂️

**Pull request closed without merging**